jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -713,6 +713,9 @@ private InterpreterResult executeSql(String propertyKey, String sql,
       if (splitQuery) {
         sqlArray = splitSqlQueries(sql);
       } else {
+        if (sql.trim().endsWith(";")) {
+          sql = sql.trim().substring(0, sql.trim().length() - 1);
+        }
```

Comment:
nit: why not sql.trim() in all other cases?

Reply:
Fixed